src/robotlibcore.py

```diff
@@ -22,16 +22,19 @@
 import inspect
 import sys
 
-try:
-    from robot.api.deco import keyword
-except ImportError:  # Support RF < 2.9
-    def keyword(name=None, tags=()):
+from robot.api.deco import keyword
+
+if not hasattr(keyword, 'robot_types'):
```

Comment:
I don't think this works. The decorator doesn't have `robot_types`, only decorated methods have. Perhaps it's easiest just to define `@keyword` here without even trying to import it anywhere.

Reply:
True.

Reply:
I went to solution that RF 3,1 is minimum version. It has been available for a pretty long time. Do not want to support too many versions.